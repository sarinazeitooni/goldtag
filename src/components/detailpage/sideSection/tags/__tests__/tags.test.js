import React from 'react';
import { shallow } from 'enzyme';
import Tags from "../tags";
describe("Tags component" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<Tags id={['','']}/>)
        expect(container).toMatchSnapshot();
    });
    it("it should render MainSwipper",()=>{
        const container = shallow(<Tags  id={['','']}/>);
        expect(container.find('button')).toBeTruthy();
    });
});