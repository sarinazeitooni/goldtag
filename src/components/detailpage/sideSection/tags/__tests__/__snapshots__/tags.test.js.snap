// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags component it should render a div 1`] = `ShallowWrapper {}`;
